src/config/cfg_rules.ini

```diff
@@ -11,7 +11,8 @@ section = ifp
 section = secrets
 section = kcm
 section_re = ^secrets/users/[0-9]\+$
-section_re = ^domain/.*$
+section_re = ^domain/[^/\@]\+$
+section_re = ^domain/[^/\@]\+/[^/\@]\+$
 section_re = ^application/.*$
 
```

Comment:
I did not want to limit any characters for application name. In domain we need to limit it in order to distinguish between trusted domain section and normal domain section.